README.md

```diff
@@ -24,3 +24,17 @@ the source code and running the tests via sbt. Some common invocations for sbt:
 * `sbt dockerComposeTest  # run the docker compose tests (slow)`
 * `sbt "dockerComposeTest -tags:DockerComposeTag" # run only tests with DockerComposeTag (while using docker)`
 * `sbt ci  # run all tests`
+
+Some tests need Kafka to run. If one prefers to run them via IDE, it's required to run the test cluster:
+```bash
+sbt dockerComposeUp
+```
+or via plain docker-compose:
+```bash
+export DOCKER_KAFKA_HOST=$(ip route get 8.8.8.8 | awk '{print $NF; exit}')
```

Comment:
I would instead have them run the `docker_setup.sh`, since this command doesn't work on macos.

Reply:
Thanks for checking that!